birdhouse/config/proxy/docker-compose-extra.yml

```diff
@@ -0,0 +1,38 @@
+version: "3.4"
+
+x-logging:
+  &default-logging
+  driver: "json-file"
+  options:
+    max-size: "50m"
+    max-file: "10"
+
+services:
+  proxy:
+    image: pavics/canarieapi:0.3.5
+    container_name: proxy
+    ports:
+      - "80:80"
+      - "443:${PROXY_SECURE_PORT}"
+      - "58094:8094"
+      - "58093:8093"
+      - "58091:8091"
+      - "58079:8079"
+      - "58086:8086"
+    volumes:
+      - ./config/proxy/conf.d:/etc/nginx/conf.d
+      - ./config/proxy/nginx.conf:/etc/nginx/nginx.conf
+      - ./config/canarie-api/docker_configuration.py:/config/docker_configuration.py
+      - ${SSL_CERTIFICATE}:/etc/nginx/cert.pem
+      - wps_outputs:/pavics-data/wps_outputs
```

Comment:
Good catch. It will be activated already if one of the birds is activated or if the wps_outputs-volume dir is added to the optional configs list manually (for some reason)

Reply:
The proxy needs to add the nginx `alias` to the mapped `wps_outputs` path dynamically if any of the WPS birds is added + needs to consider that https://github.com/bird-house/birdhouse-deploy/tree/master/birdhouse/optional-components/secure-data-proxy extends this definition if enabled as well.

Reply:
Maybe just `secure-data-proxy` missing an explicit `COMPONENT_DEPENDENCIES` to `config/wps_outputs-volume` if this config can be disabled (?). 
I think the definitions should resolve correctly the way you modified them.